main.js

```diff
@@ -5,21 +5,37 @@ let select = document.querySelector('.categories');
 let textArea = document.querySelector('.pullText');
 
 let buttonClick = document.querySelector('.addButton');
-buttonClick.addEventListener('click', () => {
-    let selectedCategory = select.options[select.selectedIndex].value;
-    let selectedDate = inputDate.value.split('-').splice(1,2).reverse().join('.');
-    let text = textArea.value;
-    let masHelp = [selectedCategory, text, selectedDate];
+let arrStorage = [];
 
+function createNewThing(needData) {
```

Comment:
эта функция не "создаёт новую штуку", а вставляет задачу в DOM. Отображение каких-либо данных где-либо, чаще всего, называется отрисовкой - rendering.

```suggestion
function renderTodoItem(item) {
```